bokeh/application/handlers/code.py

```diff
@@ -54,12 +55,8 @@ def modify_document(self, doc):
 
         module = self._runner.new_module()
 
-        # This is to prevent the module from being gc'd before the
-        # document is.  A symptom of a gc'd module is that its
-        # globals become None.
```

Comment:
Originally this was a very important comment, but now we just store modules for further use, so this comment is irrelevant. Perhaps a different one should be used instead.